packages/db/lib/config.js

```diff
@@ -23,18 +23,22 @@ class DBConfigManager extends ConfigManager {
       this.current.core.connectionString = 'sqlite://' + sqliteFullPath
     }
 
+    const versionsTableName = this.current.core.schema.length > 0
+      ? 'public.versions'
```

Comment:
The `public` schema exists only in postgres. So we should apply this default only if the DB is postgres.

Reply:
Done on 821f075